# HG changeset patch
# User Rik <rik@octave.org>
# Date 1473914006 25200
#      Wed Sep 14 21:33:26 2016 -0700
# Node ID 09c0ee94b7ee56beb94d1d5189b8ac9247ce464d
# Parent  8c920950b7ccf2974f9d84ab1700d827a3b60afd
Fix typo in cset 8c920950b7cc.

* __gnuplot_has_feature__.m: Fix typo in cset 8c920950b7cc.

diff --git a/scripts/plot/util/private/__gnuplot_has_feature__.m b/scripts/plot/util/private/__gnuplot_has_feature__.m
--- a/scripts/plot/util/private/__gnuplot_has_feature__.m
+++ b/scripts/plot/util/private/__gnuplot_has_feature__.m
@@ -24,17 +24,17 @@
 ## Author: Ben Abbott <bpabbott@mac.com>
 ## Created: 2009-01-27
 
 function res = __gnuplot_has_feature__ (feature)
   persistent features = {"minimum_version",
                          "needs_color_with_postscript",
                          "linetype",
                          "dashtype",
-                         "alphablend_linecolor"};
+                         "alphablend_linecolor",
                          "qt_terminal"};
 
   persistent has_features;
 
   if (isempty (has_features))
     try
       gnuplot_version = __gnuplot_version__ ();
     catch
